src/Store.php

```diff
@@ -132,37 +132,39 @@ public function get($key)
     }
 
     /**
-     * Store an item in the cache for a given number of minutes.
+     * Store an item in the cache for a given number of seconds.
      *
      * @param  string  $key
      * @param  mixed   $value
-     * @param  float|int  $minutes
+     * @param  float|int  $seconds
      * @return bool
      */
-    public function put($key, $value, $minutes = 0)
+    public function put($key, $value, $seconds = 0)
     {
         $val = var_export($value, true);
 
         // HHVM fails at __set_state, so just use object cast for now
-        $val = str_replace('stdClass::__set_state', '(object)', $val);
+        if (defined('HHVM_VERSION')) {
```

Comment:
References:
[Source code](https://github.com/facebook/hhvm/search?q=HHVM_VERSION)
[Usage in the wild](https://hhvm.com/blog/2393/hhvm-2-3-0-and-travis-ci#:~:text=if%20(defined(%27HHVM_VERSION%27))%20%7B%0A%20%20//%20do%20clowny%20hack%20while%20your%20issue/pull%20request%20is%20being%20worked%20on%0A%7D)